src/leo-client-app/package.json

```diff
@@ -22,7 +22,8 @@
     "moment": "^2.29.4",
     "next": "^13.5.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "suncalc": "^1.9.0"
```

Comment:
i dont think we need suncalc in the client, does it not work without? we only use suncalc in the server